compiler/src/scala/quoted/runtime/impl/QuotesImpl.scala

```diff
@@ -1439,6 +1439,8 @@ class QuotesImpl private (using val ctx: Context) extends Quotes, QuoteUnpickler
     end UnapplyTypeTest
 
     object Unapply extends UnapplyModule:
+      def apply(fun: Term, implicits: List[Term], patterns: List[Tree]): Unapply =
+        withDefaultPos(tpd.UnApply(fun, implicits, patterns, dotc.core.Symbols.defn.AnyType))
```

Comment:
I think @odersky suggested using Nothing instead of Any for the type of the unapply tree node? In any case we should use the same type in TypeAssigner and in the reflection API for consistency.

Reply:
Yes, that is part of #13302. This parameter will disappear once we rebase on #13302 or in #13302.